study/cyber-tutor/quiz/index.html

```diff
@@ -86,7 +86,7 @@
             <div class="container">
 			
                 <div style="text-align: center;">
-                    <h1 style="margin-bottom: 0;">Quiz</h1>
+                    <h1 style="margin-bottom: 0;">Cyber Quiz</h1>
```

Comment:
Use the [Bootstrap spacing helper class](https://getbootstrap.com/docs/4.0/utilities/spacing/) .mb-0 instead of inline styles for this.